wellregistry/registry/admin/monitoring_location.py

```diff
@@ -34,15 +185,29 @@ class MonitoringLocationAdminForm(ModelForm):
     """
     Registry admin form.
     """
-
-    # pylint: disable=E1101
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+        super(MonitoringLocationAdminForm, self).__init__(*args, **kwargs)
         if not self.user.is_superuser:
             user_group = _get_groups(self.user)[0]
             agency_field = ModelChoiceField(queryset=AgencyLookup.objects.filter(agency_cd=user_group),
                                             initial=user_group)
             self.fields['agency'] = agency_field
+            for item in tool_tips:
+                form_field = item['form_field']
+                self.fields[form_field].widget.attrs.update({
+                    'title': item['tooltip']
+                 })
+                widget_exists_level_2 = hasattr(self.fields[form_field].widget, 'widget')
```

Comment:
This requires some explanation. Is this needed for a particular kind of field?

Reply:
In some cases, with fields that have linked datasets instead of .widget.attrs there exists another level .widget.widget
in this case, we need to check for .widget.widget

Reply:
So the issue is with ModelChoiceFields which are implemented with the widgets.RelatedFieldWidgetWrapper. I think the code would be clear if we did something like this:
```python
if isinstance(self.fields[form_field].widget, widgets.RelatedFieldWidgetWrapper):
    widget = self.fields[form_field].widget.widget
else:
    widget = self.fields[form_field].widget

widget.attrs.update({
    'title': item['tooltip']
})
```